src/components/GenreSelect/GenreSelect.tsx

```diff
@@ -10,6 +10,9 @@ const GenreSelect = ({
   selectedGenre,
   onSelect,
 }: GenreSelectProps) => {
+  if(!selectedGenre){
```

Comment:
Not that way!
Use JS default function parameters for this purpose
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters